### DISCUSSION TITLE
分享模板 [2238900] STAR OCEAN THE SECOND STORY R [仅本体 或 +DLC 啥的]

### DISCUSSION THREAD

**Discussion by pjy612** [pjy612/SteamManifestCache#26] [Share]
[关于 内测 说明](https://github.com/pjy612/SteamManifestCache/discussions/115)

分享麻烦 统一格式
appid 用 \` 符号 包裹  
如 
\`appid:2238900\`
效果如下
`appid:2238900`

密钥 的话 使用直接 \`\`\`  进行包裹，方便别人复制。
如
\`\`\`
2238901;67cb73b183e7b5251669ae7065e69ae7e541ae57d56e19105e12323adcc2f42e
\`\`\`
效果如下
keys:
```
2238901;67cb73b183e7b5251669ae7065e69ae7e541ae57d56e19105e12323adcc2f42e
```

关于如何定位密钥 参考 wiki 
[Depot 包 库](https://github.com/pjy612/SteamManifestCache/wiki/Depot)
[DecryptionKey 密钥](https://github.com/pjy612/SteamManifestCache/wiki/DecryptionKey)
https://steamdb.info/app/2238900/depots/

如果格式没对，帖子内容 可以点右侧三个点 选 `Edit` 重新编辑
提交前 还可以切换选项卡到 `Perview` 预览效果
[便利转换脚本](https://github.com/pjy612/SteamManifestCache/discussions/60)

**上传附件请确保 上传成功后再提交**（成功的标志是 粘贴的内容 从 `Uploading...` 变为有效链接  
（可选）密钥 **额外** 做附件上传
[2238900_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13674909/2238900_config.vdf.txt)
（可选）添加与密钥匹配的 当前版本的清单（可配合 固定版本功能让旧清单也能下载游戏 方便大家）

如果 分享的 app 未被仓库收录 麻烦分享者能附加一份配套的最新清单（必要）

你们自己在 [密钥清单分享](https://github.com/pjy612/SteamManifestCache/discussions/categories/Share) 发 新的 discussion。。。

不用再下面跟。。。囧

**Comment by Scropiouos** [comment]
Darkest Dungeon® II: The Binding Blade 
最近刚出DLC所以清单可能经常要更新
```
appid:1940340
keys:
2598510;235f932f22e943699a9dd3de36d77beb99ce3e99b25eea7d1467da814906bbde
```
[2598510_6047719762198650119.zip](https://github.com/pjy612/SteamManifestCache/files/13676469/2598510_6047719762198650119.zip)

code:7260221233AF8AAE84CB85B5E65F1ACCB469F252
email:rudy_han@foxmail.com

**Reply by Scropiouos** [reply]
t.me/Scropious

**Reply by pjy612** [reply]
查收邮件

**Comment by TOP-01** [comment]
[350640] Sherlock Holmes - The Devil's Daughter 福尔摩斯：恶魔之女 [本体+DLC]
appid:350640
Key
350641;34250b46fccba475fdb0461ef0446aaf59eedee59381ce0a5552098820051f17
350642;3f2b39c40fdf1584b915fc43cd81431992f5200d4eb2171ed15c09d363679113
350643;f108b1325a22a7b771f4a23c108a309d20723fc3bcb1f2ea11901e16d47a1835
350647;1fc33e239097dc654796f7231ddcd537225891376449534226be1ddb8d0541fb
code:58A02DE597C70BEF5C62567B9391A3ECDAB19717
email:1782798059@qq.com
[Sherlock Holmes - The Devil's Daughter.zip](https://github.com/pjy612/SteamManifestCache/files/13676567/Sherlock.Holmes.-.The.Devil.s.Daughter.zip)

**Reply by pjy612** [reply]
查收邮件

**Comment by peizp** [comment]
Idol Manage
App ID	821880
"DecryptionKey"		"75915dfffeee33ee4ef1212feacc98569b9eac5bea5353d4f5a9eb96a125d1f6"
战律
App ID	607050
					"607051"
					{
						"DecryptionKey"		"f571baf67e599fe0e04cb1389a9b9f0c11a09dfdab851e7c20c3235be8102634"
					}
					"607053"
					{
						"DecryptionKey"		"c10100fabc812eed05d9fd6f7521b74def684801b032c377fbec09e9f0947c4a"

幸福工厂 
App ID	526870

depotid  526871
					"526871"
					{
						"DecryptionKey"		"ae0ab673da3c9d0f444b1cd61fe8f2235d43e6c0401684a78f4fbdfc8f0d9bc1"

code:58A02DE597C70BEF5C62567B9391A3ECDAB19717
email:957600252@qq.com

[分享.zip](https://github.com/pjy612/SteamManifestCache/files/13677249/default.zip)
第一次分享，看下对不对？

**Reply by pjy612** [reply]
查收邮件
ps.你和上面 code 一样？

**Comment by Nobody2613** [comment]
[629520] Soundpad | 本体
```
appid:629520
keys:
629521;40df4fcb0748caf1d77bff732f597eebb26b3a36539b1646d149f54eb676807f
```
code;4FA95F31FDB8F04533917E1BEF07958C1C13BDBF
email;nobody53_@outlook.com

**Reply by pjy612** [reply]
查收邮件，其实可以编辑合在一个里面

**Comment by Nobody2613** [comment]
[1787120] Sparkling Memories | 本体
```
appid:1787120
keys:
1787121;199a1bd93d00817032dd84161d51faf660feb277e2ab05a400b1bc705b33b15c
```
[1787121_manifests.zip](https://github.com/pjy612/SteamManifestCache/files/13678056/1787121_manifests.zip)

code;4FA95F31FDB8F04533917E1BEF07958C1C13BDBF
email;nobody53_@outlook.com

**Comment by Nobody2613** [comment]
[2143530] 夏日花火 Soundtrack | 本体
```
appid:2143530
keys:
2143531;393b1f6940781a245a3ce352c3acc1f1a9c4622ce5f106fd7a8e8d2e718d78a2
2143532;3ee5e1255757ee0b7f43a5f76bb82f8183c4e665c946ee8ca1108cee9e20373a
2143533;e56db559da6d1f111e36001d073b6dc5f82c221a75fd2c9fcfd24dddbbfc9e8b
```
[2143530_manifests.zip](https://github.com/pjy612/SteamManifestCache/files/13678092/2143530_manifests.zip)

code;4FA95F31FDB8F04533917E1BEF07958C1C13BDBF
email;nobody53_@outlook.com

**Reply by Nobody2613** [reply]
这游戏的ost独立成一个app了，不在游戏的depot下....

**Reply by pjy612** [reply]
问题不大，毕竟一般人不关心 ost ...

**Comment by Nobody2613** [comment]
[890720] In Other Waters | 本体
```
appid:890720
keys:
890721;f4defa9b2ccc386e447642fb9ee37eb7fc05cfe32acddef58cf5a98536cd5d4b
890722;72621cc0a911c69cc830efcf03e14979dd270e5d0f59f5a1394984b00328bf97
```
code;4FA95F31FDB8F04533917E1BEF07958C1C13BDBF
email;nobody53_@outlook.com

**Comment by chunge35** [comment]
appid:1144200
keys:
1144201;b9236894ba071d12dd117ce16220fefeb20574d7c6f7d2170e3113346867f3d6
email:chunge35@foxmail.com
code:5B9E46D8F55061729179A11F9994B349260005AD
之前那个帖子被锁了，无法回复，这个是严阵以待最新正式版的密钥清单，申请加入内测
[ready or not.zip](https://github.com/pjy612/SteamManifestCache/files/13678156/ready.or.not.zip)

**Reply by pjy612** [reply]
查收邮件，不确定锁定的帖子是否能重新编辑自己的内容

**Comment by Nobody2613** [comment]
[527100] Star Trek: Bridge Crew | 本体 + DLC
```
appid:527100
keys:
527101;31cdd603eb8d5711eece7e5604d1c692bb270a192dadf77e479179977c4600ae
806190;c2c8616a5465eca52e44891132eea8a2668392423bd2dd5d6551f1678603d14f
```
code;4FA95F31FDB8F04533917E1BEF07958C1C13BDBF
email;nobody53_@outlook.com

**Comment by Nobody2613** [comment]
[588210] HELLION | 本体
```
appid:588210
keys:
588211;5adccf972c0313b145d006cf8b973fc61d91c2bf733198915c2a8aa5d4db7baa
```
code;4FA95F31FDB8F04533917E1BEF07958C1C13BDBF
email;nobody53_@outlook.com

**Comment by Nobody2613** [comment]
[1618540] Ghost Exorcism INC | 本体
```
appid:1618540
keys:
1618541;bd23a464581d71234d60a9a93530b20637584ea6db55bf465f45c5a2869abac7
```
code;4FA95F31FDB8F04533917E1BEF07958C1C13BDBF
email;nobody53_@outlook.com

**Comment by llllllu91** [comment]
wartales 战争传说 12.14更新清单
appid:1527950
dlc:2692400
[wartales.rar.txt](https://github.com/pjy612/SteamManifestCache/files/13679579/wartales.rar.txt)

**Comment by SivanLaai** [comment]
[324140] Kalimba | 本体

```
appid:324140
keys:
324141;6beb7d65f75651a0cdf04475e51d81916fa0547db17af495899f17723e10350a
```

code;F3B3657F1B3EBEE76D6376FF80134377DC92DB22
email;[574855297@qq.com](mailto:574855297@qq.com)

[Kalimba.zip](https://github.com/pjy612/SteamManifestCache/files/13680081/Kalimba.zip)

**Reply by pjy612** [reply]
查收邮件
